cli/config/config.go

```diff
@@ -67,6 +68,9 @@ func Load(configDir string) (*configfile.ConfigFile, error) {
 	if configDir == "" {
 		configDir = Dir()
 	}
+	if strings.HasPrefix(configDir, homedir.GetShortcutString()) {
+		configDir = filepath.Join(homedir.Get(), configDir[2:])
```

Comment:
Not sure `configDir[2:]` is portable... See [Windows version of GetShortcutString](https://github.com/vdemeester/cli/blob/a9ceec72a84759c39fd7ff73cea80752d86e1e5b/vendor/github.com/docker/docker/pkg/homedir/homedir_windows.go#L23). Try with TrimPrefix instead?
```go
configDir = filepath.Join(homedir.Get(), strings.TrimPrefix(configDir, homedir.GetShortcutString()))
```

Reply:
@silvin-lubecki very good point :+1: